### ISSUE TITLE
add support for  IE 11

### ISSUE THREAD

**Issue by oliviergenevest** [LekoArts/gatsby-starter-portfolio-jodie#10]
Only the side bar is displayed, no main content. Probably because of css grid. Is there a way to add support for IE11 ? Really nice work though ;)

**Comment by LekoArts** [comment]
I think it's an issue with gatsby-image
https://github.com/gatsbyjs/gatsby/issues/4021

**Comment by oliviergenevest** [comment]
thanks, i will investigate this issue

**Comment by LekoArts** [comment]
Closing this issue as answered then. If you have questions about polyfilling certain features or browser support in general please open up an issue on the gatsby repo as more people will find answers there then. Maybe we also need to improve on some things :)

Thanks for asking!

**Comment by oliviergenevest** [comment]
I opened the demo page with Edge (Windows 10). The site was only showing on a small part of the screen (the first column of the grid used on the homepage). I found out how to fix the problem, related to grid css and not to Gatsby image as you mentioned earlier, this worked for me :
1) changing the  `<main> ` tag by a  `<div role='main'> ` tag, grid doesn't work well with  `<main> ` tag with MS Edge.
2) Apply   `grid-column-start:2;` to this div.

It works like a charm after that. Hope it helps.
